legacy-server/index.js

```diff
@@ -0,0 +1,80 @@
+const express = require('express');
```

Comment:
Is there a reason you're keeping the old code around? Is it possible you could just leave it in a "legacy" branch?

Reply:
No reason in particular, will remove it now.